src/alignof.cc

```diff
@@ -0,0 +1,68 @@
+
```

Comment:
Okay! I don't have much experience with the C++ anyways, but perhaps someone else may have an opinion on it.